text/0061-error-format-and-definitions.md

```diff
@@ -0,0 +1,807 @@
+
+### Context
+
+This error occurs during the dump creation process. The dump creation was interrupted for various reasons.
+
+💡 Errors returned asynchronously in a `dump` object, does not include a definition of the http code.
+
+### Error Definition
+
+```json
+{
+    "message": "The creation of the dump :dumpId failed due to :reason.",
+    "code": "dump_process_failed",
+    "type": "internal",
+    "link": "https://docs.meilisearch.com/errors#dump_process_failed"
+}
+```
+
+- The `:reason` is inferred when the message is generated.
+
+---
+
+## database_size_limit_reached
+
+### Context
+
+This error occurs when the user tries to add documents and the maximum size of the database reaches the limit. The user can correct this error by increasing the database size limit.
+
+### Error Definition
+
+HTTP code `500 Internal Server Error`
+
+```json
+{
+    "message": "The maximum size of :databaseSizeLimit for the database has been reached.",
+    "code": "database_size_limit_reached",
+    "type": "internal",
+    "link": "https://docs.meilisearch.com/errors#database_size_limit_reached"
+}
+```
+
+- The `:databaseSizeLimit` is inferred when the message is generated. e.g. `100GiB`
+
+---
+
+## no_space_left_on_device
+
+### Context
+
+This error occurs when the host system partition has reached its maximum capacity and is no longer accepting writes.
+
+### Error Definition
+
+HTTP Code: `500 Internal Server Error`
+
+```json
+{
+    "message": "There is no more space left on the device. Consider increasing the size of the disk/partition.",
+    "code": "no_space_left_on_device",
+    "type": "internal",
+    "link": "https://docs.meilisearch.com/errors#no_space_left_on_device"
+}
+```
+
+---
+
+## invalid_store_file
+
+### Context
+
+This error occurs when the data.ms folder is in an inconsistent state. It can happen for various reasons. An .mdb file can be corrupted, the data.ms folder has been replaced by a file...
+
+### Error Definition
+
+HTTP Code: `500 Internal Server Error`
+
+```json
+{
+    "message": "The database file is in an invalid state.",
+    "code": "invalid_store_file",
+    "type": "internal",
+    "link": "https://docs.meilisearch.com/errors#invalid_store_file"
+}
+```
+
+---
+
+# auth type
+
+## missing_authorization_header
+
+### Context
+
+This error occurs when the route is protected and the `X-MEILI-API-KEY` header is not provided.
+
+### Error Definition
+
+HTTP Code: `401 Unauthorized`
+
+```json
+{
+    "message": "The X-MEILI-API-KEY header is missing.",
+    "code": "missing_authorization_header",
+    "type": "auth",
+    "link": "https://docs.meilisearch.com/errors#missing_authorization_header"
+}
+```
+
+---
+
+## invalid_api_key
+
+### Context
+
+This error occurs when the route is protected and the value of the `X-MEILI-API-KEY` header does not allow access to the resource.
+
+### Error Definition
+
+HTTP Code: `403 Forbidden`
+
+```json
+{
+    "message": "The provided API Key is invalid.",
```

Comment:
FYI: @meilisearch/docs-team has been avoiding using `API Key` to avoid confusion between `master`, `private` and `public` keys. We have discussed using authentication key when any of the keys will do.

I'm not sure whether this applies here, so for the moment I'm just noting it for the moment.